examples/llm/sft/hf.py

```diff
@@ -75,17 +75,9 @@ def squad(tokenizer) -> pl.LightningDataModule:
         grad_clip = None
     use_dist_samp = False
 
-    model = llm.HfAutoModelForCausalLM(args.model)
+    model = llm.HfAutoModelForCausalLM(model_name=args.model, model_accelerator=args.model_accelerator)
```

Comment:
can we pass a partial function instead?

Reply:
I also thought about it but I am just not sure how it'll look from the user perspective.